config/install/field.field.paragraph.stat.field_stat_prefix.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_stat_prefix
+    - paragraphs.paragraphs_type.stat
+id: paragraph.stat.field_stat_prefix
+field_name: field_stat_prefix
+entity_type: paragraph
+bundle: stat
+label: Prefix
+description: 'Include a symbol immediately before the stat value. Will not be animated.'
```

Comment:
🤔 just a thought, will this only ever be a symbol or could be it something like "approx." (and vice versa, "people" perhaps, to be "approx. 10000 people")?

Reply:
Yeah, I think that would be useful, if you have the time :)